src/Asset/AssetManager.php

```diff
@@ -188,55 +197,125 @@ class AssetManager
      * }
      * ```
      */
-    public $hashCallback;
+    private $hashCallback;
+
+    /**
+     * @var bool whether to use symbolic link to publish asset files. Defaults to false, meaning asset files are copied
+     *           to {@see basePath}. Using symbolic links has the benefit that the published assets will always be
+     *           consistent with the source assets and there is no copy operation required. This is especially useful
+     *           during development.
+     *
+     * However, there are special requirements for hosting environments in order to use symbolic links. In particular,
+     * symbolic links are supported only on Linux/Unix, and Windows Vista/2008 or greater.
+     *
+     * Moreover, some Web servers need to be properly configured so that the linked assets are accessible to Web users.
+     * For example, for Apache Web server, the following configuration directive should be added for the Web folder:
+     *
+     * ```apache
+     * Options FollowSymLinks
+     * ```
+     */
+    private $linkAssets = false;
```

Comment:
We can set it true by default. Windows versions w/o symlinks are rare, MacOS has the support as well.